src/middleware/README.md

```diff
@@ -0,0 +1,3 @@
+# Middleware
+
+**[Astro supports middleware](https://docs.astro.build/en/guides/middleware/) that allows you to intercept requests and responses and inject behaviors dynamically every time a page or endpoint is about to be rendered.**
```

Comment:
@sjoerdbeentjes you recently mentioned the caveat that when home resolves as a static file, middleware will not run. Perhaps it is wise to explicitly add this here.

Reply:
Very specific caveat. I don't think people will catch that just by a line in the docs.